src/utils/formatTags.ts

```diff
@@ -0,0 +1,11 @@
+export function formatTags(tags: string[]): string[] {
+  const actualArray = tags[0];
```

Comment:
full or both parts array might be better name? Not sure as to why the threshold is two? think naming could be slightly improved here